src/test/java/teammates/test/cases/datatransfer/FeedbackResponseAttributesTest.java

```diff
@@ -92,6 +94,34 @@ public void testSetResponseDetails_shouldDoDeepCopy() {
 
     }
 
+    @Test
+    public void testGetJsonString_withQuestionDetails_shouldSerializeToConcreteQuestionDetailsClass() {
```

Comment:
On a relevant note. The point is to test the new adaptor `TeammatesFeedbackResponseDetailsAdapter`. So I guess the test should reside in `JsonUtilsTest.java`?